R/theme-elements.r

```diff
@@ -130,6 +131,38 @@ element_text <- function(family = NULL, face = NULL, colour = NULL,
   )
 }
 
+#' @param colour_1 geom accent colour 1,
+#'   typically a lighter version of colour
+#' @param colour_2 geom accent colour 2,
+#'   typically a bright colour used for geom_smooth et al.
+#' @param fill_1 geom accent fill colour, typically a darker version of fill
+#' @param ... other accepted spellings of themable aesthetics, e.g. "color"
+#' @export
+#' @rdname element
+element_geom <- function(fill = NULL,
+                         fill_1 = NULL,
+                         colour = NULL,
+                         colour_1 = NULL,
+                         colour_2 = NULL,
```

Comment:
I would add arguments `color`, `color_1`, `color_2` and use those if the `colour` versions are `NULL` (or the other way round, doesn't really matter).